src/renderer/components/server-modal-form.jsx

```diff
@@ -314,7 +314,7 @@ export default class ServerModalForm extends Component {
           </div>
           <div className={`four wide field ${this.highlightError('database')}`}>
             <label>Database/Keyspace</label>
-            <div className="ui action input">
+            <div className={this.state.client === "sqlite" && "ui action input"}>
```

Comment:
On could also change it to 
```class={this.state.client === "sqlite" ? "ui action input" : "ui input"}```
which would keep the semantic declaration scheme. 
Don't know, if it's necessary though.